### PR TITLE
ADD - Ship station integration

### PR DESCRIPTION
**Pull request by sromic**
Added ShipStation integration

### CODE REVIEW

File: sources/catalog/shipStation/config.json [hunk 1]
```diff
@@ -0,0 +1,70 @@
+{
+  "title": "ShipStation",
+  "description": "Shipping tracking",
+  "apiVersion": "2020-03-02",
+  "platform": "shipStations",
```
Comment (paulkr) [inline sources/catalog/shipStation/config.json:5]: Platform should be `shipstation`

File: sources/catalog/shipStation/config.json [hunk 2]
```diff
@@ -0,0 +1,70 @@
...
+  "category": "tracking",
+  "image": "https://assets.buildable.dev/catalog/node-templates/shipStation.svg",
```
Comment (paulkr) [inline sources/catalog/shipStation/config.json:7]: https://assets.buildable.dev/catalog/node-templates/shipstation.svg

File: sources/catalog/shipStation/config.json [hunk 3]
```diff
@@ -0,0 +1,70 @@
...
+      "label": "Enter your ShipStation Api Key",
+      "placeholder": "15975fdsaasd"
+    },
+    {
+      "name": "SHIP_STATION_SECRET_KEY",
+      "label": "Enter your ShipStation Secret Key",
+      "placeholder": "75945fdsaf97"
+    }
+  ],
+  "eventSchema": {
```
Comment (paulkr) [inline sources/catalog/shipStation/config.json:21]: Is this all the data available for a sample event payload?

File: sources/catalog/shipStation/config.json [hunk 4]
```diff
@@ -0,0 +1,70 @@
...
+    "subscriptionDelayMultiplier": 0.0
+  },
+  "paths": {
+    "id": "_.body.WebHookID",
+    "event": "_.body.resource_type",
+    "payload": "_.body",
+    "secret": null,
+    "signature": null
+  },
+  "connectionTypes": ["source", "action"],
```
Comment (paulkr) [inline sources/catalog/shipStation/config.json:37]: The value here should just be `["source"]`

File: sources/catalog/shipStation/shipStation.ts [hunk 5]
```diff
@@ -0,0 +1,236 @@
+import axios, { AxiosInstance, AxiosResponse } from "axios";
```
Comment (paulkr) [inline sources/catalog/shipStation/shipStation.ts:1]: This file name should be in all lowercase: `shipstation.ts`

**Review by paulkr — CHANGES_REQUESTED** [review]
Looks great so far!

File: sources/catalog/shipStation/config.json (hunk 3 above)
Comment (sromic) [reply sources/catalog/shipStation/config.json:26]: Yes, 
payload for all events looks like the same, having only two properties:

e.g.

```
{
  "resource_url": "https://ssapi11.shipstation.com/shipments?batchId=123456&includeShipmentItems=True",
  "resource_type": "ITEM_SHIP_NOTIFY"
}
```

Also, headers do not include any specific/useful information, e.g.

```
connection    close
accept-encoding    gzip, deflate
content-length    139
host    webhook.site
content-type    application/json
accept    application/json, application/xml, text/json, text/x-json, text/javascript, text/xml
x-newrelic-transaction    PxRTUgUFXAtWBlZWA1RSA1MJFB8EBw8RVU4aBgtbUlYLA1tWBVVWBwEBD0NKQVkHBAZTUAUHFTs=
x-newrelic-id    VQMHVVRSGwABU1NRBgYGXw==
```

**Pull request merged**